playbooks/byo/openshift-cluster/redeploy-certificates.yml

```diff
@@ -14,7 +14,36 @@
   gather_facts: no
   tasks:
   - include_vars: ../../byo/openshift-cluster/cluster_hosts.yml
+  - set_fact:
+      openshift_deployment_type: "{{ deployment_type }}"
```

Comment:
Any particular reason to use `set_fact` here instead of `vars`? vars has a lower runtime performance impact.